src/stylesheets/core/_system-tokens.scss

```diff
@@ -142,18 +142,18 @@ Layout grid widths
 
 // prettier-ignore
 $system-layout-grid-widths: (
-  1: 100% * 1 / 12,
-  2: 100% * 2 / 12,
-  3: 100% * 3 / 12,
-  4: 100% * 4 / 12,
-  5: 100% * 5 / 12,
-  6: 100% * 6 / 12,
-  7: 100% * 7 / 12,
-  8: 100% * 8 / 12,
-  9: 100% * 9 / 12,
-  10: 100% * 10 / 12,
-  11: 100% * 11 / 12,
-  12: 100% * 12 / 12,
+  1: divide(100% * 1, 12),
+  2: divide(100% * 2, 12),
+  3: divide(100% * 3, 12),
+  4: divide(100% * 4, 12),
+  5: divide(100% * 5, 12),
+  6: divide(100% * 6, 12),
+  7: divide(100% * 7, 12),
+  8: divide(100% * 8, 12),
+  9: divide(100% * 9, 12),
+  10: divide(100% * 10, 12),
+  11: divide(100% * 11, 12),
+  12: divide(100% * 12, 12),
```

Comment:
Let's just confirm this gives the expected results.

Reply:
A small difference in that there's no rounding for the final precision value at 10 decimal places (e.g. `16.6666666667%` vs. `16.6666666666%`):

https://www.sassmeister.com/gist/aduth/bcfcde6eeeec8272e56c4321a6678f48

Reply:
It _might_ be possible to emulate the rounding: Do one extra level of recursion past `$precision`, then `+ 1` the last digit if that value is `>= 5`. 🤔

Reply:
Here's a version with rounding: https://www.sassmeister.com/gist/aduth/95bacec332c6390bacaff9fe5ccc82b9

Reply:
Pushed the rounding changes in dad2a78.